examples/tailscale-vpn/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM alpine:latest
+
+RUN apk update \
+    && apk add curl openrc \
+    && rm -rf /var/cache/apk/*
+
+RUN curl -fsSL https://tailscale.com/install.sh | sh
+
+COPY . .
+
+# Run on container startup.
+RUN chmod +x /app/start.sh
+CMD ["/app/start.sh"]
```

Comment:
Not sure I like the way this is being done. There are tailscale docker images, and I'm pretty sure another Docker layer could be added (ie, two "FROM"s).

Reply:
This automatically fetches the latest tailscale version, and doesn't need updating from anyone else, as it would if it were to use a tailscale docker image.

I don't think there is a point in a separate layer, as this literally just installs the necessary dependencies for tailscale to run. It's extremely lightweight, and it already uses alpine.